--- src/Makefile.am.link	2010-02-03 18:36:30.000000000 +0100
+++ src/Makefile.am	2010-02-03 18:36:41.000000000 +0100
@@ -76,7 +76,7 @@
 endif
 
 libpst_la_SOURCES     = $(common_source) $(common_header)
-libpst_la_LIBADD      = $(LTLIBICONV)
+libpst_la_LIBADD      = $(LTLIBICONV) -pthread
 
 EXTRA_DIST =
 if !NEED_XGETOPT
--- src/Makefile.in.link	2010-02-03 18:36:47.000000000 +0100
+++ src/Makefile.in	2010-02-03 18:37:00.000000000 +0100
@@ -323,7 +323,7 @@
 
 @STATIC_TOOLS_FALSE@libpst_la_LDFLAGS = $(NO_UNDEFINED) -version-info @LIBPST_VERSION_INFO@
 libpst_la_SOURCES = $(common_source) $(common_header)
-libpst_la_LIBADD = $(LTLIBICONV)
+libpst_la_LIBADD = $(LTLIBICONV) -pthread
 EXTRA_DIST = $(am__append_4)
 
 # set the include path found by configure
