diff -Nru a/src/Makefile.am b/src/Makefile.am
--- a/src/Makefile.am	2018-08-01 17:15:48.000000000 +0200
+++ b/src/Makefile.am	2018-08-26 14:28:10.034569178 +0200
@@ -80,7 +80,7 @@
 endif
 
 libpst_la_SOURCES     = $(common_source) $(common_header)
-libpst_la_LIBADD      = $(LTLIBICONV)
+libpst_la_LIBADD      = $(LTLIBICONV) -lpthread -lz
 
 EXTRA_DIST =
 if !NEED_XGETOPT
diff -Nru a/src/Makefile.in b/src/Makefile.in
--- a/src/Makefile.in	2018-08-01 17:16:00.000000000 +0200
+++ b/src/Makefile.in	2018-08-26 14:28:46.484567645 +0200
@@ -355,7 +355,7 @@
 
 @STATIC_TOOLS_FALSE@libpst_la_LDFLAGS = $(NO_UNDEFINED) -version-info @LIBPST_VERSION_INFO@
 libpst_la_SOURCES = $(common_source) $(common_header)
-libpst_la_LIBADD = $(LTLIBICONV)
+libpst_la_LIBADD = $(LTLIBICONV) -lpthread -lz
 EXTRA_DIST = $(am__append_4)
 
 # set the include path found by configure
